<launch>
    <!-- Lane Keeping System -->
    <node name="Lane_Keeping_System" pkg="lane_keeping_system" type="lane_keeping_system_node" output="screen"/>
    <param name="config_path" type="str" value="$(find lane_keeping_system)/config/config.yaml"/>
    
    <!-- Xycar Motor -->
    <include file="$(find xycar_motor)/launch/xycar_motor.launch"/>
    
    <!-- USB Cam -->
    <include file="$(find usb_cam)/launch/usb_cam-noviewer.launch"/>

    <!-- Yolo Module -->
    <include file="$(find yolov3_trt_ros)/launch/yolov3_trt_ros_detection.launch"/>
</launch>
